cmd/lakectl/cmd/root.go

```diff
@@ -195,6 +213,37 @@ var rootCmd = &cobra.Command{
 	},
 }
 
+var excludeStatsCmds = []string{
+	"lakectl_doctor",
+	"lakectl_config",
+}
+
+func sendStats(ctx context.Context, client api.ClientWithResponsesInterface, cmd string) {
+	resp, err := client.PostStatsEventsWithResponse(ctx, api.PostStatsEventsJSONRequestBody{
+		Events: []api.StatsEvent{
+			{
+				Class: "lakectl",
+				Name:  cmd,
+				Count: 1,
+			},
+		},
+	})
+
+	var (
+		errStr     string
+		statusCode string
+	)
+	if err != nil {
+		errStr = err.Error()
+	}
+	if resp != nil && resp.StatusCode() != http.StatusNoContent {
+		statusCode = strconv.Itoa(resp.StatusCode())
+	}
+	if errStr != "" || statusCode != "" {
+		_, _ = fmt.Fprintf(os.Stderr, "Warning: failed sending statistics! status code: %s error: %s\n", statusCode, errStr)
+	}
```

Comment:
Simplified version of the above - the Status() will require the code and the text.

```suggestion
	var errStr string
	if err != nil {
		errStr = err.Error()
	} else if resp.StatusCode() != http.StatusNoContent {
		errStr = resp.Status()
	}
	if errStr != "" {
		_, _ = fmt.Fprintf(os.Stderr, "Warning: failed sending statistics: %s\n", errStr)
	}
```